R/provider-bedrock.R

```diff
@@ -54,18 +59,24 @@ chat_aws_bedrock <- function(
   model = NULL,
   profile = NULL,
   api_args = list(),
-  echo = NULL
+  echo = NULL,
+  endpoint = NULL
 ) {
   check_installed("paws.common", "AWS authentication")
   cache <- aws_creds_cache(profile)
   credentials <- paws_credentials(profile, cache = cache)
-
+  
   model <- set_default(model, "anthropic.claude-3-5-sonnet-20240620-v1:0")
   echo <- check_echo(echo)
 
+  # Clean endpoint URL if provided
+  if (!is.null(endpoint) && nzchar(endpoint)) {
+    endpoint <- sub("/$", "", endpoint)
+  }
```

Comment:
I don't think this is necessary as `req_url_path_append()` will take care of any extra slashes.